### PR TITLE
add preview tags to test

### PR DESCRIPTION
**Pull request by gokul-sol**
### Jira link

See [PROJ-XXXXXX](https://tools.hmcts.net/jira/browse/PROJ-XXXXXX)

### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

**Review by pats-john — DISMISSED** [review]
Assuming that we test the preview-regression as part of this PR...

File: README.md [hunk 1]
```diff
@@ -192,6 +192,14 @@ If you need to run the CCD provider test against the Preview environment for a p
     @VersionSelector(tag = "feature_branch_name")})
 
 ```
+
+### Running Functional UI tests against Preview env on a Pipeline
+
+By default preview pipelines would just have 1 test as part of sanity check on Preview env, but QA's would need to add label `test-suite:preview-regression` to their respective ticket PR's to enable running regression UI test pack against preview env.
+
+In order to include Bundling & Hearings tests as part of preview pipeline, QA's need to add an additional label `test-suite:aat-regression` against their PR. As a pre-requisite to add "aat-regression" label please make sure your PR has gone through the config set-up explained in this section [Running tribunals with hearings enabled](https://github.com/hmcts/sscs-tribunals-case-api/tree/enable-preview-test-coverage?tab=readme-ov-file#running-tribunals-with-hearings-enabled) of readme.
```
Comment (sanjayparekh-hmcts) [inline README.md:200]: The linked url is wrong - need to point to some section on this (current) page

File: definitions/test/e2e/enhanced-confidentiality.spec.ts [hunk 2]
```diff
@@ -22,7 +22,7 @@ test.describe("Enhanced confidentiality test", async() => {
         await createBundleSteps.triggerBundleForConfidentialCase();
     });
 
-    test("Refuse - confidentiality request for a party on a case", {tag: ['@nightly-pipeline']}, async({ uploadResponseSteps, enhancedConfidentialitySteps }) => {
+    test("Refuse - confidentiality request for a party on a case", {tag: ['@preview-regression','@nightly-pipeline']}, async({ uploadResponseSteps, enhancedConfidentialitySteps }) => {
```
Comment (sanjayparekh-hmcts) [inline definitions/test/e2e/enhanced-confidentiality.spec.ts:25]: do you need to mention the preview-regression tag in the readme too?

**Review by sanjayparekh-hmcts — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (gokul-sol) [reply README.md:200]: Will check the url

File: definitions/test/e2e/enhanced-confidentiality.spec.ts (hunk 2 above)
Comment (sanjayparekh-hmcts) [reply definitions/test/e2e/enhanced-confidentiality.spec.ts:25]: fine, but its still not clear in the readme that its the test-suite:<labels> you add in github which picks up specific tests according to <labels>

**Review by sanjayparekh-hmcts — DISMISSED** [review]

**Review by sanjayparekh-hmcts — APPROVED** [review]

**Review by pats-john — APPROVED** [review]

**Review by sanjayparekh-hmcts — APPROVED** [review]

**Pull request merged**